spark-connector/v3.3/spark/build.gradle.kts

```diff
@@ -144,10 +144,14 @@ tasks.test {
 
   val skipITs = project.hasProperty("skipITs")
   val skipSparkITs = project.hasProperty("skipSparkITs")
+  val skipSparkSQLITs = project.hasProperty("skipSparkSQLITs")
```

Comment:
Do we need a configuration to enable or disable this test?

Reply:
I feel like there are too many configurations to control whether to run something or not, do we have better solutions?

I would incline to use gradle to control whether to run this or not, by default we would not run these SQL tests.